src/librustc_plugin_impl/registry.rs

```diff
@@ -33,9 +29,6 @@ pub struct Registry<'a> {
     #[doc(hidden)]
     pub krate_span: Span,
 
-    #[doc(hidden)]
-    pub syntax_exts: Vec<NamedSyntaxExtension>,
-
     #[doc(hidden)]
     pub llvm_passes: Vec<String>,
```

Comment:
Aside: is this used for anything? (If not, maybe we should remove it in different PR?)

Reply:
Yeah, I've seen it, the `krate_span` field above also seems unused.
I'll investigate this and make a separate PR later (next weekend perhaps).